src/modules/metadata.js

```diff
@@ -91,3 +91,37 @@ export const getDynamicTimeDimensionsMetadata = (
         return acc
     }, {}),
 })
+
+export const isPopulatedObject = (input) => {
```

Comment:
Could you add unit tests for both of these new functions?

Reply:
Yes will do. Actually `transformMetaDataResponseObject` is not really new. This is simply some logic that was extracted from the component because we now need to do the same thing in two places.